test/unit/js/persist.test.js

```diff
@@ -285,6 +300,21 @@ test("persist : save state data by key, get state data by key", function() {
 	deepEqual(clonedState, this.data);
 });
 
+$.each([null, undefined, ''], function(i, v) {
+	test("persist : remove state data with value by key", function() {
+		// Given
+		var state = $.persist("TESTKEY");
+		$.persist("TESTKEY", this.data);
+
+		// When
+		var clonedState = $.persist("TESTKEY", v);
+
+		// Then
+		equal(clonedState, v);
+	});
+})
```

Comment:
There is duplicated tests intentionally.
One is for mocking test for headless test,
the other is for testing native browser environment.
```
module("persist: mock", {
```
```
module("persist: native", {
```